### PR TITLE
update(cmake,userspace): bumped to libs master.

### PR DESCRIPTION
**Pull request by FedeDP**
**What type of PR is this?**

/kind cleanup

**Any specific area of the project related to this PR?**

/area build
/area engine

**What this PR does / why we need it**:

Bump to latest libs master (6c11056815b9eff787c69f9b2188a2ae503533c9)

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
update(build,userspace): bumped libs and driver to 6c11056815b9eff787c69f9b2188a2ae503533c9
```

### CODE REVIEW

File: userspace/falco/app/actions/configure_interesting_sets.cpp [hunk 1]
```diff
@@ -177,8 +167,7 @@ falco::app::run_result falco::app::actions::configure_interesting_sets(falco::ap
 	}
 
 	s.selected_sc_set.clear();
-	s.selected_tp_set.clear();
-	
+
```
Comment (FedeDP) [inline userspace/falco/app/actions/configure_interesting_sets.cpp:170]: We don't have a `tracepoint` API anymore. We use PPM_SC for everything.
Tracepoints are already extracted by the ruleset, just like syscalls.

**Comment by FedeDP** [comment]
/cc @jasondellaluce @incertum @Andreagit97 
Do we need to enforce some tracepoint, like signaldeliver?
In [libs 0.10.x](https://github.com/falcosecurity/libs/blob/6b98d411cd635480fda0cb4c9e860165aa70d7de/userspace/libscap/scap.c#L1038) we were enforcing:
* sys_enter
* sys_exit
* SCHED_PROC_EXIT
* SCHED_SWITCH
* SCHED_PROC_FORK
* SCHED_PROC_EXEC

Now, sched switch is only added if one wants to retrieve `sched_switch` event.
`sys_enter`, `sys_exit`, `sched_proc_fork` and `sched_proc_exec` are correctly managed by [driver/ppm_tp.c](https://github.com/falcosecurity/libs/blob/master/driver/ppm_tp.c#L40).
I think we just need to enforce `SCHED_PROC_EXIT` somewhere, right?

**Comment by LucaGuerra** [comment]
Thank you for this @FedeDP !

**Comment by FedeDP** [comment]
I break API stuff, i upgrade :laughing:

**Comment by FedeDP** [comment]
About `sched_proc_exit`: we enforce `sinsp_state_sc_set` that in turns enforces `scap_get_modifies_state_ppm_sc`, that enforces the `procexit` ppm_sc given that it modifies state. 
We are ok!

**Comment by jasondellaluce** [comment]
>About sched_proc_exit: we enforce sinsp_state_sc_set that in turns enforces scap_get_modifies_state_ppm_sc, that enforces the procexit ppm_sc given that it modifies state.
We are ok!

We should make sure this is also included in the set repair logic implemented by Melissa, so that those essential events are collected even on custom syscall base sets.

**Review by Andreagit97 — APPROVED** [review]
/approve

**Review by jasondellaluce — APPROVED** [review]
/approve

**Pull request merged**